src/wp-includes/functions.php

```diff
@@ -6068,11 +6068,11 @@ function wp_trigger_error( $function_name, $message, $error_level = E_USER_NOTIC
 	$message = wp_kses(
 		$message,
 		array(
-			'a' => array( 'href' ),
-			'br',
-			'code',
-			'em',
-			'strong',
+			'a'      => array( 'href' ),
```

Comment:
Is this right? I believe it should rather be:

```suggestion
			'a'      => array( 'href' => true ),
```

Reply:
This is required. Error strings having `<a href="/"></a>` are being stripped out without your suggestion.

Reply:
Humm, in core it shows it is supposed to be `'href' => true`:

https://github.com/WordPress/wordpress-develop/blob/5115c7ac80d7260b73d467f784b790587d61210b/src/wp-includes/kses.php#L68-L71

Reply:
Oh, you applied this suggestion in the end anyway?

Reply:
First I added a test case https://github.com/WordPress/wordpress-develop/pull/6671/commits/afbacc045b298ee5a112ba44ab02d4f772d21951 which turned out to be failing, so added the suggested fix https://github.com/WordPress/wordpress-develop/pull/6671/commits/7b0029cb3c90f993dd01e0a8327fc8df5d24eb4a.

Reply:
Oh, I misread what you wrote. I thought you said _with_ my suggestion not _without_ 😊